### PR TITLE
media-fonts/source-han-sans: Version bump to 2.000 and 2.001

### PR DESCRIPTION
**Pull request by shimataro**
create v2.000 and v2.001 (copy of v1.004)

### CODE REVIEW

**Comment by shimataro** [comment]
@rossbridger 
There is no special reason for it, so v2.000 can be removed safely :smiley:

File: media-fonts/source-han-sans/source-han-sans-2.000.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (ajakk) [inline media-fonts/source-han-sans/source-han-sans-2.000.ebuild:1]: I think adding 2.000 is unnecessary if 2.001 is added too.

File: media-fonts/source-han-sans/source-han-sans-2.001.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (ajakk) [inline media-fonts/source-han-sans/source-han-sans-2.001.ebuild:4]: Please update to EAPI=7.

File: media-fonts/source-han-sans/source-han-sans-2.001.ebuild [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+inherit font
```
Comment (ajakk) [inline media-fonts/source-han-sans/source-han-sans-2.001.ebuild:5]: Newline between EAPI and inherit lines, please.

File: media-fonts/source-han-sans/source-han-sans-2.001.ebuild [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	l10n_zh-CN? ( https://github.com/adobe-fonts/${PN}/raw/${PV}R/SubsetOTF/SourceHanSansCN.zip -> ${PN}-zh_CN-${PV}.zip )
+	l10n_zh-TW? ( https://github.com/adobe-fonts/${PN}/raw/${PV}R/SubsetOTF/SourceHanSansTW.zip -> ${PN}-zh_TW-${PV}.zip )"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~x64-macos"
+IUSE="l10n_ja l10n_ko +l10n_zh-CN l10n_zh-TW"
+REQUIRED_USE="|| ( l10n_ja l10n_ko l10n_zh-CN l10n_zh-TW )"
+
+S=${WORKDIR}
```
Comment (ajakk) [inline media-fonts/source-han-sans/source-han-sans-2.001.ebuild:25]: Should quote "${WORKDIR}"

File: media-fonts/source-han-sans/source-han-sans-2.001.ebuild [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+FONT_SUFFIX="otf"
+RESTRICT="binchecks strip"
+
+src_install() {
+	use l10n_ja && FONT_S="${S}/SourceHanSansJP" font_src_install
+	use l10n_ko && FONT_S="${S}/SourceHanSansKR" font_src_install
```
Comment (ajakk) [inline media-fonts/source-han-sans/source-han-sans-2.001.ebuild:31]: Can we add the Hong Kong thing as mentioned in the bug? https://bugs.gentoo.org/699610

**Review by ajakk — CHANGES_REQUESTED** [review]
You can also add a `Closes: https://bugs.gentoo.org/699610` to your commit message. Also, we will need the signoff to contain your real name according to [GLEP-76](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin)

**Comment by ajakk** [comment]
I guess no one is there :(

**Pull request closed without merging**